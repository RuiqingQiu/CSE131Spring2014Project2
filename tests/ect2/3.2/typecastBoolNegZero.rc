function : int main(){
   auto f = 0.0;
   cout << *(int*)&f << endl;
   cout << (bool)(*(int*)&f) << endl;
   cout << (bool)f << endl;
   f = -f;
   cout << *(int*)&f << endl;
   cout << (bool)(*(int*)&f) << endl;
   cout << (bool)f << endl;
   
   return 0;
}
