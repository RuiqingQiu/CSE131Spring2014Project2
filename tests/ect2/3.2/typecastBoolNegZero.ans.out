0
false
false
-2147483648
true
false
